docs/grid/theming/theming-variables.md

```diff
@@ -1,3 +1,7 @@
+---
+sidebar_label: "Variables"
```

Comment:
I would stick to `Theming` or `Style by CSS` 

`Variables` are not descriptive enough